web/client/components/mapcontrols/searchcoordinates/CoordinatesSearch.js

```diff
@@ -108,6 +139,7 @@ const CoordinatesSearch = ({
     onZoomToPoint,
     onChangeCoord,
     defaultZoomLevel,
+    currentMapCRS,
```

Comment:
This seems to be missing in
https://github.com/geosolutions-it/MapStore2/pull/10220/files#diff-455b07539c76115f6dce6c982a88461c05ae2ab7cb8a2a3fac5a2a6ad58ba9cc